### DISCUSSION TITLE
Documentation to develop an ORDERS section like in the demo

### DISCUSSION THREAD

**Discussion by KONEY** [filamentphp/filament#5643] [Help]
Hello!

I'd like to build an ORDERS admin section, just like the one from Filament Demo. Can anyone point me in which part of the documentation I should start? I have already a working admin for other section but I need one specifically like "orders".

Thanks!

**Comment by danharrin** [comment]
The source code is available on filamentphp/demo

**Reply by emincer** [reply]
Came here looking for this. I need that exact same feature. Oh man, there's going to be so much to learn in this demo code. Thanks Dan.
